src/com/tony/server/Main.java

```diff
@@ -38,12 +38,15 @@ public static void main(String[] args) {
                 uriList.add("/" + listOfFiles[i].getName());
             }
         }
+        uriList.remove("/.DS_Store");
+        uriList.remove("/form");
+        uriList.remove("/method_options");
         return uriList;
```

Comment:
It's up to the customer. If they request the ability to hide specific files in the public folder then you should create a story for it. For now, remove that code and if you think it's an important feature then bring it up at the next IPM.